Automattic/woocommerce-payments
# Update the payment details page to reflect the Tap to Pay transactions channel

Fixes #5765

#### Changes proposed in this Pull Request
Replaces the `In-Person` Channel field with `Tap to Pay on iPhone` or `Tap to Pay on Android` for Tap to Pay transactions.

<img width="771" alt="Screenshot 2023-03-17 at 18 52 06" src="https://user-images.githubusercontent.com/8667118/225970289-f87581be-d40e-4c42-a9b9-09fd6bb72f4b.png">

#### Testing instructions
- Create an order and pay it with Tap to Pay
- Navigate to the Transactions list page
- Navigate to Tap to Pay transactions details page
- Verify a the Channel column has the `Tap to Pay on iPhone` value.

*

-------------------


**Post merge**
